build.sbt

```diff
@@ -21,7 +21,7 @@ lazy val commonSettings = Seq(
   organization := "com.cognite",
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
-  version := "2.0.8-SNAPSHOT",
+  version := "2.0.9",
```

Comment:
@vuhainguyen this isn't right, 2.0.8-SNAPSHOT means we're working on 2.0.8 which will be the next release. So you should change this to 2.0.8 for the release, then bump to 2.0.9-SNAPSHOT.